Plugins/ExternalTexture/Source/ExternalTexture_Shared.h

```diff
@@ -38,39 +38,40 @@ namespace Babylon::Plugins
         arcana::make_task(context.BeforeRenderScheduler(), arcana::cancellation_source::none(),
             [&context, &runtime, deferred = std::move(deferred), impl = m_impl]()
         {
-            auto handle = bgfx::createTexture2D(impl->Width(), impl->Height(), impl->HasMips(), 1, impl->Format(), impl->Flags());
+            // REVIEW: The bgfx texture handle probably needs to be an RAII object to make sure it gets clean up during the asynchrony.
+            //         For example, if any of the schedulers/dispatches below don't fire, then the texture handle will leak.
```

Comment:
Could you address this with an `expected`-handling cleanup continuation?

Reply:
No, on shutdown, the continuation may never happen right?